### DISCUSSION TITLE
Marlin Babystepping bug, seems important?

### DISCUSSION THREAD

**Discussion by Reggi3** [Jyers/Marlin#458] [Q&A] [answered]
https://github.com/MarlinFirmware/Configurations/issues/466

Looks like it could impact a lot of people, checked all of the configuration_adv.h files in the Configuration Files folder, all of them have the same BABYSTEP_MULTIPLICATOR_Z 40 define.  Except for the one in 'Default Marlin Config'.

**Comment by Jyers** [comment, marked as answer]
Certainly interesting. I think I'll wait to see if the marlin devs make the change. Thanks for pointing this out!

**Reply by Reggi3** [reply]
You're welcome!! From the answer, it looks like they're waiting for a pull request... in the meantime, I'm personally going to test and see if setting it to 4 makes a difference?  I think 40steps = .1mm, given that the z offset stuff is handled in increments of 0.01mm, 4 seems like a better bet than 1? doesn't seem much point in 0.0025mm increments if we can't actually access that on screen?

**Reply by Jyers** [reply]
Well let me know if you find any difference. It also may not effect the menu options due to the fact that I send the babystep commands in terms of mm, which may override this value.

**Reply by Reggi3** [reply]
Out of curiosity and never shy of asking questions with obvious answers, what is the resolution that you're using for babystepping?  is it 0.01mm by any chance?  If so then, as you say, it probably won't affect anything because you're effectively already doing that BABYSTEP_MULTIPLICATOR_Z 4 in mm.
